pages/Documents/RichMessaging/WhatsAppBusinessMessagesTemplates/product-messages-template.md

```diff
@@ -0,0 +1,263 @@
+---
+pagename: Product Messages Template
+redirect_from:
+Keywords: structured content rich messaging whatsapp business messages product messages
+sitesection: Documents
+categoryname: "Rich Messaging"
+documentname: Viber Templates
+permalink: whatsapp-business-messages-templates-product-messages-template.html
+indicator: messaging
+---
+
+### Overview
+
+WhatsApp now supports Product Messaging! Brands can now send single or multiple product messages to their WhatsApp Messenger customers.
+This allows the customer to view the product(s), ask questions about given products as well as purchase and complete the order 
```

Comment:
instead of "purchase and complete the order" please say "submit a shopping cart"